src/main/kotlin/org/rust/ide/inspections/SelfConventionInspection.kt

```diff
@@ -15,7 +15,7 @@ class SelfConventionInspection : RustLocalInspectionTool() {
             override fun visitImplMethodMember(m: RustImplMethodMember) {
                 val convention = SELF_CONVENTIONS.find { m.identifier.text.startsWith(it.prefix) } ?: return
                 if (m.selfType !in convention.selfTypes) {
-                    holder.registerProblem(m.selfArgument ?: m.identifier, convention)
+                    holder.registerProblem(m.fnParamsWithSelf.selfArgument ?: m.identifier, convention)
```

Comment:
Is it possible to rename this property to `params` or something? The only method I have found is to add `methods=[...]` directive in grammar, but this doesn't remove the obscure version.